.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: python
+python: 2.7
+env:
+    - TOX_ENV=py26
+    - TOX_ENV=py27
+    - TOX_ENV=py32
```

Comment:
py32 seems to fail because of unicode literals in pip. I think we can drop it from it.